railties/test/generators/app_generator_test.rb

```diff
@@ -955,16 +955,20 @@ def test_skip_webpack_install
     end
 
     generator([destination_root], skip_webpack_install: true).stub(:rails_command, command_check) do
-      quietly { generator.invoke_all }
+      generator.stub :bundle_command, nil do
+        quietly { generator.invoke_all }
+      end
     end
 
     assert_gem "webpacker"
     assert_no_file "config/webpacker.yml"
 
     output = Dir.chdir(destination_root) do
-      `rails --help`
+      `bin/rails help`
```

Comment:
Nice throwback to https://github.com/rails/rails/pull/38789 😉

Reply:
I'm sure there are plenty more, gotta catch 'em all 😂